packages/wrangler/src/index.tsx

```diff
@@ -1234,18 +1249,13 @@ function createCLIParser(argv: string[]) {
                 args["experimental-enable-local-persistence"] || false
               }
               accountId={config.account_id}
-              assetPaths={getAssetPaths(
-                config,
-                args["experimental-public"] || args.site,
-                args.siteInclude,
-                args.siteExclude
-              )}
+              assetPaths={assetPaths}
               port={args.port || config.dev.port || (await getLocalPort())}
               ip={args.ip || config.dev.ip}
               inspectorPort={
                 args["inspector-port"] ?? (await getInspectorPort())
               }
-              public={args["experimental-public"]}
+              isWorkersSite={Boolean(args.site || config.site)}
```

Comment:
Are we certain that if site is an empty string we want to fallback on `config.site`?

Reply:
We can't pass an empty string in the cli, and it's not a valid value anyway. It's fine